business_requirement/models/business.py

```diff
@@ -269,7 +270,8 @@ def name_get(self):
         result = []
         for br in self:
             if br.ref:
-                formatted_name = u'[{}] {}'.format(br.ref, br.description)
+                formatted_name = u'[{}][{}] {}'.format(br.ref, br.name,
```

Comment:
Yes you are right. I think It should be like, right? 

`formatted_name = u'[{}] {}'.format(br.ref, br.name, br.description)`

Will do the needful, thanks!